features/keychain/api/index.d.ts

```diff
@@ -1,9 +1,20 @@
 import KeyIdentifier from '@exodus/key-identifier'
+import { UnsignedTransaction } from '@exodus/tx-signer'
```

Comment:
```suggestion
import type { UnsignedTransaction } from '@exodus/tx-signer'
```

Reply:
same for other import

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/132/commits/901dea305df329c55708e3e9ff2aab8e0785c50e